tests/large_test/khc/post_test.cpp

```diff
@@ -102,5 +104,5 @@ TEST_CASE( "HTTP Post" ) {
     REQUIRE( res == KHC_ERR_OK );
     REQUIRE( on_read_called == 2 );
     REQUIRE( on_header_called > 1 );
-    REQUIRE( on_write_called == 2 );
+    REQUIRE( on_write_called >= 1 );
```

Comment:
This change should be OK. By debugging with `printf("ON_WRITE %s", buffer);` it seems now everything is received in just one call. The test was depending on a behavior that cannot be guaranteed.